x-pack/test/functional/apps/transform/creation/index_pattern/creation_index_pattern.ts

```diff
@@ -582,6 +620,15 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
               );
             }
 
+            await transform.testExecution.logTestStep('opens field stats flyout from agg input');
+            const aggInputFieldStatsEntries = testData.fieldStatsEntries
+              ? testData.fieldStatsEntries.filter((e) => e.isAggInput)
+              : [];
+
+            for (const { fieldName, type } of aggInputFieldStatsEntries) {
+              await transform.wizard.clickAggInputFieldStatTrigger(fieldName, type);
+            }
```

Comment:
Not exactly sure if that will solve the flaky test issue, but I think it would contribute to overall test stability if we closed the flyout once we're done with assertions, e.g. something like this:
```suggestion
            for (const { fieldName, type } of aggInputFieldStatsEntries) {
              await transform.wizard.clickAggInputFieldStatTrigger(fieldName, type);
              await transform.wizard.closeFieldStatsFlyout(); // not implemented yet
            }
```
If we do it like this, we need to add it to all the places where we open the flyout for assertions, but I'll not repeat this review comment for all the places :wink: